src/QorusIntegration.ts

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+import httpsAxios from './utils/httpsAxios';
+import { CatchAll } from './utils/catchDecorators';
+import log from 'logger-decorator';
+import LocalStorageManager from './utils/LocalStorageManager';
+
+export interface ILoginParams {
+  user: string;
+  pass: string;
+}
+
+enum ApiPaths {
```

Comment:
Maybe we could support multiple API versions, so instead of this being hardcorded to `api/latest` it should be something like `${API_VERSION}` where the default is `latest`